tests/examples.lean

```diff
@@ -113,62 +113,26 @@ end refine_struct
 
 /- traversable -/
 
-@[derive traversable]
+attribute [derive traversable, derive is_lawful_traversable] list option
+-- do not put in instances because they are not universe polymorphic
+
+@[derive traversable, derive is_lawful_traversable]
```

Comment:
Could we have `derive is_lawful_traversable` automatically derive `traversable` and `functor` and `is_lawful_functor` if they aren't already there?

Reply:
Yes, `@[derive traversable, derive is_lawful_traversable]` actually implies `functor` and `is_lawful_functor`. I could actually write `@[derive is_lawful_traversable]` to do all four but I thought it might be a bit too concise.

Reply:
you can also derive multiple things by writing `@[derive [traversable, is_lawful_traversable]]`

Reply:
That's really cool! Thanks!